testnets/nomictestnet/chain.json

```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../chain.schema.json",
```

Comment:
need two (up) directories to reference the schema's location.
"../../chain.schema.json"